epam-promo/components/pickers/PickerModal.tsx

```diff
@@ -87,10 +99,10 @@ export class PickerModalImpl<TItem, TId> extends PickerModalBase<TItem, TId> {
                         search={ this.lens.prop('dataSourceState').prop('search').toProps() }
                         showSearch={ false }
                         rows={ rows }
-                        renderNotFound={ this.props.renderNotFound && (() => this.props.renderNotFound({
+                        renderNotFound={ this.props.renderNotFound ? (() => this.props.renderNotFound({
```

Comment:
move it into separate function, because now we will have new prop on each render